From 57e447b2da9740ff0cb1c9e5c1ba230eb211010f Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sun, 3 Aug 2025 22:43:49 -0400
Subject: [PATCH 1/2] Fixup installation process

---
 visualc/make-install-vngspice.bat | 22 +++++++++-------------
 1 file changed, 9 insertions(+), 13 deletions(-)

diff --git a/visualc/make-install-vngspice.bat b/visualc/make-install-vngspice.bat
index 6894d84..8e8847e 100644
--- a/visualc/make-install-vngspice.bat
+++ b/visualc/make-install-vngspice.bat
@@ -1,19 +1,18 @@
-@echo off
+@echo on
 
-REM copy ngspice.exe, codemodels *.cm to C:\Spice or Spice64
+REM copy ngspice.exe or ngspice.dll, codemodels *.cm to %LIBRARY_PREFIX%
 REM arguments to make-install-vngspiced:
-REM %1: path to ngspice.exe, %2, %3: fftw or 64 (64 bit)
+REM %1: path to ngspice.exe or ngspice.dll, %2, %3: fftw or 64 (64 bit)
 
 if "%2" == "64" goto b64
 if "%3" == "64" goto b64
 
-set dst=c:\Spice
+set dst=%LIBRARY_PREFIX%
 set cmsrc=.\codemodels\Win32\Release
 
 mkdir %dst%\bin
 mkdir %dst%\lib\ngspice
 
-copy "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.OPENMP\vcomp140.dll" %dst%\bin\
 copy %cmsrc%\analog.cm %dst%\lib\ngspice\analog.cm
 copy %cmsrc%\digital.cm %dst%\lib\ngspice\digital.cm
 copy %cmsrc%\table.cm %dst%\lib\ngspice\table.cm
@@ -24,23 +23,21 @@ copy %cmsrc%\spice2poly.cm %dst%\lib\ngspice\spice2poly.cm
 if "%2" == "fftw" goto copy2
 if "%3" == "fftw" goto copy2
 
-copy %1\ngspice.exe %dst%\bin\
+copy %1 %dst%\bin\
 goto end
 
 :copy2
-copy %1\ngspice.exe %dst%\bin\
-copy ..\..\fftw-3.3-dll32\libfftw3-3.dll %dst%\bin\
+copy %1 %dst%\bin\
 goto end
 
 :b64
 
-set dst=c:\Spice64_KLU
+set dst=%LIBRARY_PREFIX%
 set cmsrc=.\codemodels\x64\Release
 
 mkdir %dst%\bin
 mkdir %dst%\lib\ngspice
 
-copy "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x64\Microsoft.VC140.OPENMP\vcomp140.dll" %dst%\bin\
 copy %cmsrc%\analog64.cm %dst%\lib\ngspice\analog.cm
 copy %cmsrc%\digital64.cm %dst%\lib\ngspice\digital.cm
 copy %cmsrc%\table64.cm %dst%\lib\ngspice\table.cm
@@ -51,12 +48,11 @@ copy %cmsrc%\spice2poly64.cm %dst%\lib\ngspice\spice2poly.cm
 if "%2" == "fftw" goto copy2-64
 if "%3" == "fftw" goto copy2-64
 
-copy %1\ngspice.exe %dst%\bin\
+copy %1 %dst%\bin\
 goto end
 
 :copy2-64
-copy %1\ngspice.exe %dst%\bin\
-copy ..\..\fftw-3.3-dll64\libfftw3-3.dll %dst%\bin\
+copy %1 %dst%\bin\
 
 :end
 mkdir %dst%\share\ngspice\scripts\src\ngspice
-- 
2.48.1

